packages/mattermost-redux/src/selectors/entities/emojis.ts

```diff
@@ -11,9 +11,17 @@ import {IDMappedObjects} from 'mattermost-redux/types/utilities';
 
 import {createIdsSelector} from 'mattermost-redux/utils/helpers';
 
+export const getCustomEmojisEnabled: (state: GlobalState) => boolean = createSelector(
```

Comment:
As mentioned, this doesn't really need to be wrapped in `createSelector`. It could just be
```typescript
export const getCustomEmojisEnabled(state: GlobalState) {
    return getConfig(state)?.EnableCustomEmoji === 'true';
}
```

Reply:
✅  Updated as per request/discussion